autoreject/autoreject.py

```diff
@@ -1218,26 +1218,56 @@ def __init__(self, bad_epochs, labels, ch_names):
         assert len(bad_epochs) == labels.shape[0]
         assert len(ch_names) == labels.shape[1]
 
-    def plot(self):
-        """Plot."""
+    def plot(self, orientation='vertical', show=True):
+        """Plot.
+
+        Parameters
+        ----------
+        orientation : 'vertical' or 'horizontal'
+            If `'vertical'`, will plot sensors on x-axis and epochs on y-axis.
+            If `'horizontal'`, will plot epochs on x-axis and sensors
+            on y-axis.
+        show : bool
+            If True, display the figure immediately.
+
+        Returns
+        -------
+        figure : Instance of matplotlib.figure.Figure
+        """
         import matplotlib.pyplot as plt
 
-        plt.figure(figsize=(12, 6))
-        plt.imshow(self.labels, cmap='Reds',
-                   interpolation='nearest')
-        # XXX to be fixed
-        ch_names_ = self.ch_names[7::10]
+        figure = plt.figure(figsize=(12, 6))
```

Comment:
```suggestion
        figure, ax = plt.subplots(figsize=(12, 6))
```

then no need for plt.gca()

Reply:
right, done.